prov/sockets/src/sock_fabric.c

```diff
@@ -768,7 +768,7 @@ SOCKETS_INI
 
 	fi_param_define(&sock_prov, "pe_affinity", FI_PARAM_STRING,
 			"If specified, bind the progress thread to the indicated range(s) of Linux virtual processor ID(s). "
-			"This option is currently not supported on OS X. Usage: id_start[-id_end[:stride]][,]");
+			"This option is currently not supported on OS X and Windows. Usage: id_start[-id_end[:stride]][,]");
```

Comment:
Please move this change to the other patch where sockets related changes are added.

Reply:
Dona